app/src/hooks/LocalUserAttributesHelperComponent.js

```diff
@@ -17,6 +19,8 @@ const LocalUserAttributesHelperComponent = () => {
     if (!userAttributes?.deviceId) {
       const deviceId = uuidv4();
       stableDispatch(globalActions.updateUserAttributes({ deviceId }));
```

Comment:
Should we directly fire the `submitAttrUtil(APP_CONSTANTS.GA_EVENTS.ATTR.CUSTOM_DEVICE_ID, userAttributes?.deviceId);` here only as local.ts does the same thing